compiler-rt/lib/scudo/standalone/allocator_config.def

```diff
@@ -104,6 +99,11 @@ PRIMARY_OPTIONAL(const bool, EnableContiguousRegions, true)
 // guarantee a performance benefit.
 PRIMARY_OPTIONAL_TYPE(ConditionVariableT, ConditionVariableDummy)
 
+// Defines the type and scale of a compact pointer. A compact pointer can
```

Comment:
`Defines` ... `and scale` -> the scale is actually defined by `CompactPtrScale`

I think it might help here to document how to choose the proper type, for instance
> This type is used to efficiently store arrays of pointers to free blocks, and it must be large enough to express offsets within an entire region, i.e. this type's maximum possible value must uphold `MaxCompactPtrValue << CompactPtrScale >= 1 << RegionSizeLog`.  The default type (`uptr`) trivially satisfies this requirement but, depending on `CompactPtrScale` and `RegionSizeLog`, a smaller type could work too.